DESCRIPTION

```diff
@@ -114,3 +114,4 @@ Collate:
     'utilities-molecule-plots.R'
     'utilities-theme.R'
     'utils.R'
+Remotes: github::Open-Systems-Pharmacology/OSPSuite.RUtils
```

Comment:
hmm, I would not do this as well. 
It would install the **development** version of ospsuite.utils, which in the most cases is NOT what the user wants 
(user usually needs to install a **released** version instead)

Reply:
The package release mechanics would be the following:
- in `develop` branch so ongoing dev, have `Remotes::user/pkg` that target dev version of dependencies.
- For a release, this same line is changed to  `Remotes::user/pkg@*release` to target the latest release available on github.
- Then the code is changed back to `Remotes::user/pkg` to pursue dev.

This way:
1. When a user install from a release he always get the **released** dependencies.
2. When a dev installs a dev version, he gets the dev version of dependencies.

As the different OSP packages are heavily depending on each others and are developed in tandem, making point 2 automatic is significantly safer. 
This also allow the user to install the packages in a easier way without needing to maintain links to release on our side. (btw the link for installing {ospsuite.utils} is out of date in the package website).

Together with this, ones usually:
A. Update the NEWS file
   - For release: Increment the NEWS section with the version number,
   - After release: Add a new header `# pkg (development version)

B. Update the version number
  - For release: match the version with what was provided by AppVeyor and remove the `.9000`,
  - After release: add the `.9000` again.

See these two commits for a real example:
- https://github.com/tidyverse/tidyr/commit/561fb8eeab6eb229f84646a1123890775846c978
- https://github.com/tidyverse/tidyr/commit/d4b730a858b4bbdd3ffd4250250e982b160fddc8

Reply:
>Do we have specific rules regarding when releases are made ? 

tlf and utils are usually released when it's required by the new release of either ospsuite-R or ospsuite.reportingengine (RE).
ospsuite-R is usually released with the new OSP release plus on-demand releases in between (e.g. when important new features are available) or if it's required by the new release of RE.
RE is released when all features planned for the next release are implemented.

>But they should, right ? At least within OSP universe.

Not nesesserily. E.g. if we have a release of RE, which relies on the tlf V2; then we create tlf release V3 which is not compatible with V2 but there is no need for the new RE release. Then the latest RE does not work with the latest tlf.